resources/views/emails/forgotPasswordEmail.blade.php

```diff
@@ -10,6 +10,6 @@
 
 		Welcome to Room-rent app, {{$user->name}}!!!!! <br>
 		Please follow the following link to reset your password:
-		<a href="http://roomrent.dev/api/forgotPasswordForm/{{$user->email}}/{{$user->forgotPasswordToken}}">Reset Your Password</a>
+		<a href="http://roomrent.dev/api/showForgotPasswordForm/{{$user->email}}/{{$user->forgotPasswordToken}}">Reset Your Password</a>
```

Comment:
This is a wrong practice. Please use concept of named route. eg `<a href={{ route('route.name', $paramteres) }} `